tests/queries_tests/tag_test.js

```diff
@@ -0,0 +1,41 @@
+import chai, { expect } from 'chai'
+import * as tag from '../../src/database/queries/tag'
+
+describe('tag', () => {
+
+  const fakeTags = [
+    {
+      names: 'Massage'
+    },
+    {
+      names: 'Spa'
+    }
+  ]
+
+  beforeEach( () =>
+    Promise.all([
+      tag.deleteAll(),
+      tag.add(fakeTags)
+    ])
+  )
+
+  it('should exist', () =>
```

Comment:
I would never omit the `{}` of an arrow function unless it was both:

- a one-liner
- the implicitly returned value is being used

you should put curlies here